packages/xo-web/src/xo-app/backup-ng/restore/index.js

```diff
@@ -161,9 +167,10 @@ export default class Restore extends Component {
           first = backup
         }
         count[backup.mode] = (count[backup.mode] || 0) + 1
+        size = backup.size
```

Comment:
Backups sizes can be different. We need to display the total size here and the size per backup on the modal.